google-cloud-contrib/google-cloud-nio/pom.xml

```diff
@@ -112,6 +112,18 @@
               <shadedPattern>shaded.cloud-nio.org</shadedPattern>
             </relocation>
             <relocation>
+              <pattern>io</pattern>
```

Comment:
I thought the only thing we really wanted to export was com.google.cloud. Isn't a general shading pattern a good thing, then?